src/test/java/parser/JsonParserTest.java

```diff
@@ -0,0 +1,69 @@
+package parser;
+
+import java.io.File;
+
+import org.junit.jupiter.api.Disabled;
+import shop.Cart;
+import shop.RealItem;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class JsonParserTest {
+    final String testCartName = "anna-cart";
+    final String testCartFileName = "./src/main/resources/" + this.testCartName + ".json";
+    Cart testCart;
+
+
+    @org.junit.jupiter.api.BeforeEach
+    void setUp() {
+        testCart = new Cart(testCartName);
+
+        RealItem bag = new RealItem();
+        bag.setName("Bag");
+        bag.setPrice(22.9);
+        bag.setWeight(1.5);
+
+        testCart.addRealItem(bag);
+    }
+
+    @org.junit.jupiter.api.AfterEach
+    void tearDown() {
+        File f = new File(testCartFileName);
+        if (f.exists())
+        {
+            f.delete();
+        }
+        testCart = null;
+    }
+
+    @org.junit.jupiter.api.Test
+    void writeToFile() {
+        Parser parser = new JsonParser();
+        parser.writeToFile(testCart);
+
+        File f = new File(testCartFileName);
+        assertTrue(f.exists());
```

Comment:
Always add message on failure for assertTrue and assertFalse. It is required, otherwise you will see "expected true, but found false" on test failure and it will be not clear exactly what is wrong